staging/src/kubevirt.io/client-go/kubecli/vmi.go

```diff
@@ -462,14 +462,14 @@ func (v *vmis) VSOCK(name string, options *v1.VSOCKOptions) (StreamInterface, er
 func (v *vmis) SEVFetchCertChain(name string) (v1.SEVPlatformInfo, error) {
 	sevPlatformInfo := v1.SEVPlatformInfo{}
 	uri := fmt.Sprintf(vmiSubresourceURL, v1.ApiStorageVersion, v.namespace, name, "sev/fetchcertchain")
-	err := v.restClient.Get().RequestURI(uri).Do(context.Background()).Into(&sevPlatformInfo)
+	err := v.restClient.Get().AbsPath(uri).Do(context.Background()).Into(&sevPlatformInfo)
```

Comment:
Spotted one more issue: all SEV* methods use `context.Background()`, but I think the context should be passed from the caller, as in all other APIs. But it's not related to this PR. Can be fixed in a follow-up.

Reply:
Sure! Well, I can address it here if you agree! Just let me know

Reply:
Would be great. Thanks!

Reply:
@vasiliy-ul I added some commits to address the missing context. PTAL

Reply:
Thanks a lot! Looks good!